src/pykeen/losses.py

```diff
@@ -530,6 +533,24 @@ def forward(
         ))
 
 
+# class SoftMarginRanking(MarginRankingLoss):
```

Comment:
these don't necessarily need subclasses since the superclass takes care of all of them, but this PR does introduce the `DeltaPointwiseLoss` base class shared between the softpluss loss and pointwise hinge loss. Maybe we should break out a parent class for this and make these variants explicit?